### PR TITLE
Move body of ScriptTask::handle_mouse_move_event into a method on

### PR DESCRIPTION
**Pull request by gurobokum**
Document #4984

### CODE REVIEW

File: components/script/script_task.rs [hunk 1]
```diff
@@ -986,7 +985,17 @@ impl ScriptTask {
...
             MouseUpEvent(..) => {}
             MouseMoveEvent(point) => {
-                self.handle_mouse_move_event(pipeline_id, point);
+                let page = get_page(&*self.page.borrow(), pipeline_id);
+                let frame = page.frame();
+                let document = frame.as_ref().unwrap().document.root();
+                let mouse_over_targets = &mut *self.mouse_over_targets.borrow_mut();
+
+                if document.r().handle_mouse_move_event(self.js_runtime.ptr, point, mouse_over_targets) {
+
```
Comment (kmcallister) [inline components/script/script_task.rs]: Nit: don't need a blank line here

File: components/script/dom/document.rs [hunk 2]
```diff
@@ -450,6 +452,91 @@ impl<'a> DocumentHelpers<'a> for JSRef<'a, Document> {
             None => {}
         }
     }
+
+    fn handle_mouse_move_event (self, js_runtime: *mut JSRuntime, point: Point2D<f32>, mouse_over_targets: &mut Option<Vec<JS<Node>>>) -> bool{
```
Comment (kmcallister) [inline components/script/dom/document.rs]: Nit: space before `(`

Comment (kmcallister) [inline components/script/dom/document.rs]: Also can you wrap this line, and the line added in the `trait`, to fit within 100 columns (ideally, 80)?

**Comment by kmcallister** [comment]
r=me with those syntax tweaks

**Comment by JIoJIaJIu** [comment]
Can you give me advise how to wrap func, please?

**Comment by jdm** [comment]
Just move arguments that go beyond 80 characters on to the next line, starting in the same column as the arguments on the previous line.

**Comment by JIoJIaJIu** [comment]
Omg, excuse me)

**Comment by JIoJIaJIu** [comment]
> In order to help critic track your changes, please do not make in-place history rewrites (e.g. via git rebase -i or git commit --amend) 
But when im trying to make `pull ustream master` i've got merge conflict.
What is the best way to avoid this? Thx

**Comment by jdm** [comment]
Ah, the best solution there is to use branches when making commits. I leave master tracking the upstream master, and rebase my feature branches based on it.

**Comment by jdm** [comment]
You could do
```
git reset --hard 1eef07c
git checkout -b move_fn
git checkout master
git reset --hard f0b5794e44a2a28212aaf3fac4c340a3f32a8515
```
and then you should have a `master` branch that can be pulled from upstream again. You'll need to make a new pull request for the new `move_fn` branch, though.

**Comment by JIoJIaJIu** [comment]
thank you @jdm

**Pull request closed without merging**